### PR TITLE
Update docs of DMFF

### PR DESCRIPTION
**Pull request by Roy-Kid**
Xinyan and I have set up a framework of docs and updated how to contribute code to DMFF. We have discussed the content of the docs but we still don't know whether others can understand what we are talking about. I agree that we need to introduce an external developer to find the problem in the docs.

### CODE REVIEW

File: docs/user_guide/xml_spec.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
+# How to write XML file
```
Comment (jameswind) [inline docs/user_guide/xml_spec.md:1]: Translate to English?

File: docs/admp/theory.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+# 理论背景
```
Comment (jameswind) [inline docs/admp/theory.md:1]: Translate to English?

**Comment by Ericwang6** [comment]
@Roy-Kid The docs is rendered with `mkdoc`? Do we have a rendered page of the docs? perhaps a html link:)

**Comment by Ericwang6** [comment]
@Roy-Kid I think I can just made changes on your code directly?

**Comment by Roy-Kid** [comment]
> @Roy-Kid I think I can just make changes on your code directly?

Certainly. It depends on how you manage git.

**Pull request closed without merging**